src/app/core/components/left-nav/left-nav-group-datasource.ts

```diff
@@ -14,7 +14,12 @@ export class LeftNavGroupDataSource extends LeftNavDataSource<GroupInfo, NavTree
   }
 
   addDetailsToTreeElement(contentInfo: GroupInfo, treeElement: NavTreeElement): NavTreeElement {
-    return contentInfo.title ? { ...treeElement, title: contentInfo.title } : { ...treeElement };
+    let group = treeElement;
+    if (contentInfo.title) group = { ...treeElement, title: contentInfo.title };
+    if (contentInfo.navData) group = { ...treeElement,
+      children: contentInfo.navData.children.map(mapChild)
+    };
+    return group;
```

Comment:
I think the local variable `group` is useless here, this is equivalent to:
```ts
if (contentInfo.navData) return { ...treeElement, children: … }
if (contentInfo.title) return { ...treeElement, title: … }
return treeElement
```

Reply:
if it has both, only the last if populates the group

Reply:
ah of course, that was a bug. Fixed.